src/Service/UserBlocker.php

```diff
@@ -34,7 +34,7 @@ public function __construct( MediawikiApi $api ) {
 	 * @throws InvalidArgumentException
 	 * @return bool
 	 */
-	public function block( $user, BlockOptions $options = null ) {
+	public function block( $user, BlockOptions $options ) {
```

Comment:
Looks okay!

Could also leave this as it was and instead have an :+1: 

if( is_null( $options ) ) {
   $options = new BlockOptions();
}